### PR TITLE
Conveyor Sprite Addition: Directional Markers & Bugfix & Small Tweaks

### PR DESCRIPTION
**Pull request by VaelophisNyx**
# Document the changes in your pull request

Conveyors were Very irritating in that they're hard to tell which direction they're moving in. This solves that, and also a bug I noticed when testing.
Adjusts the anchored conveyor and switch description to mention you can change their properties with specific tools
Makes conveyors 1000 iron instead of 3000 so cargo deciding to factorio doesn't drain resources so hard
Adds a price of 50 iron to conveyor switches so they can be recycled properly

# Spriting

These sprites are exceptionally hard to show off or even export so I'm just going to have to ask you to check sprite diffs-
8 directional state animations are pain.

# Changelog

:cl:  
bugfix: inverting an active conveyor now updates the sprite properly
tweak: conveyor sprites now feature a directional indicator light
tweak: conveyors now only cost 1000 iron instead of 3000
tweak: conveyor switches cost 50 iron now
spellcheck: conveyors and conveyor switches now tell you what tools do what to them in the description
imageadd: added proper sprite state for inverted conveyors that are off
/:cl:

### CODE REVIEW

**Review by KillerOrcaCora — APPROVED** [review]
They are telling me to look at bot diff? Who DO THEY THINK THEY ARE!

File: code/modules/recycling/conveyor2.dm [hunk 1]
```diff
@@ -361,7 +362,8 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	desc = "A conveyor control switch assembly."
 	icon = 'icons/obj/recycling.dmi'
 	icon_state = "switch-off"
-	w_class = WEIGHT_CLASS_BULKY
+	w_class = WEIGHT_CLASS_SMALL
```
Comment (JamieD1) [inline code/modules/recycling/conveyor2.dm]:
```suggestion
	w_class = WEIGHT_CLASS_BULKY
```

File: code/modules/recycling/conveyor2.dm [hunk 2]
```diff
@@ -394,10 +396,10 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	desc = "A conveyor belt assembly."
 	icon = 'icons/obj/recycling.dmi'
 	icon_state = "conveyor_construct"
-	max_amount = 30
+	max_amount = 100
```
Comment (JamieD1) [inline code/modules/recycling/conveyor2.dm]:
```suggestion
	max_amount = 30
```

File: code/modules/recycling/conveyor2.dm [hunk 3]
```diff
@@ -394,10 +396,10 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
...
 	singular_name = "conveyor belt"
-	w_class = WEIGHT_CLASS_BULKY
-	materials = list(/datum/material/iron = 3000)
+	w_class = WEIGHT_CLASS_NORMAL
```
Comment (JamieD1) [inline code/modules/recycling/conveyor2.dm]:
```suggestion
	w_class = WEIGHT_CLASS_BULKY
```

File: code/modules/recycling/conveyor2.dm [hunk 4]
```diff
@@ -394,10 +396,10 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
...
+	materials = list(/datum/material/iron = 500)
```
Comment (JamieD1) [inline code/modules/recycling/conveyor2.dm]:
```suggestion
	materials = list(/datum/material/iron = 1000)
```

File: code/modules/recycling/conveyor2.dm [hunk 5]
```diff
@@ -433,6 +435,8 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 /obj/item/stack/conveyor/thirty
 	amount = 30
 
+/obj/item/stack/conveyor/onehundred //I am so tired of having incomplete stacks from auotlathes making 5 less than the max stack.
+	amount = 100
```
Comment (JamieD1) [inline code/modules/recycling/conveyor2.dm]:
```suggestion
```

**Review by JamieD1 — DISMISSED** [review]
Im Sorry, but Conveyors cause lag at the moment, until the new subsystem is in, I cant allow people to make factorio style shit, I dont want to encourage it with this PR, once its merged you can make these changes in a new PR.

Also your title is misleading, its not small tweaks

File: code/modules/recycling/conveyor2.dm (hunk 4 above)
Comment (VaelophisNyx) [reply code/modules/recycling/conveyor2.dm]: this one has to be here, currently these things munch on the station's iron supply way too hard

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**